lib/bundle-validator.js

```diff
@@ -66,15 +69,38 @@ BundleValidator.prototype.sizeOf = function (path, callback) {
     return sizeOf(path, callback);
 };
 
+/**
+ * If theme schema exists we need to validate it make sure it passes all defined checks.
+ * @param callback
+ * @returns {*}
+ */
+function validateThemeSchema(callback) {
```

Comment:
Seems like there is a lot of commonality with `validateThemeConfiguration`. Perhaps a refactor is in order?